src/data/ons_pd_may_19.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, ons_pd_csv_path, load_data=True):
             "County": self.settings["Boundaries folder"] + r"Counties_and_Unitary_Authorities_December_2017_Generalised_Clipped_Boundaries_in_UK\Counties_and_Unitary_Authorities_December_2017_Generalised_Clipped_Boundaries_in_UK.shp",
             "Ward": self.settings["Boundaries folder"] + r"Wards_May_2019_Boundaries_UK_BSC\Wards_May_2019_Boundaries_UK_BSC.shp",
             "PCon": self.settings["Boundaries folder"] + r"Westminster_PCON_Dec_2017_Generalised_Clipped_UK\Westminster_Parliamentary_Constituencies_December_2017_Generalised_Clipped_Boundaries_in_the_UK.shp",
-            "LSOA": self.settings["Boundaries folder"] + r"Lower_Layer_Super_Output_Areas_December_2011_Boundaries_EW_BGC\Lower_Layer_Super_Output_Areas_December_2011_Boundaries_EW_BGC.shp",
+            "LSOA": self.settings["Boundaries folder"] + r"Lower_Layer_Super_Output_Areas_December_2011_Boundaries_EW_BSC\Lower_Layer_Super_Output_Areas_December_2011_Boundaries_EW_BSC.shp",
```

Comment:
perhaps at some point we should have options for which base geographies (i.e. generalisation) to use?

Reply:
Good enhancement idea :)